java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -2003,4 +2003,21 @@ void testStringTitlize() {
       assertColumnsAreEqual(expected, result);
     }
   }
+
+  @Test
+  void testNansToNulls() {
+    try (ColumnVector cvFloat = ColumnVector.fromBoxedFloats(1.2f, Float.POSITIVE_INFINITY,
+        Float.NEGATIVE_INFINITY, null, Float.NaN, Float.MAX_VALUE, Float.MIN_VALUE, 435243.2323f);
+         ColumnVector cvDouble = ColumnVector.fromBoxedDoubles(1.2d, Double.POSITIVE_INFINITY,
```

Comment:
There should be values of other bit representation for NaN (-NaN and at least one other IEEE 754 NaN value that isn't the canonical NaN value produced by Float.NaN).